doc/automatic-tls.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "Experimental: Automatic TLS"
+docpage = true
+[menu.docs]
+    parent = "automatic-tls"
++++
+
+As of [Conduit v0.5.0][conduit-v0.5.0], Conduit can be configured to automatically
+instrument applications to communicate with Transport Layer Security (TLS).
+
+When TLS enabled, Conduit automatically establishes and authenticates
+secure,private connections between Conduit proxies. This is done without
+breaking unencrypted communication with endpoints that are not configured
+with TLS-enabled Conduit proxies.
+
+This feature is currently **experimental** and is designed to _fail open_ so
+that it cannot easily break existing applications. As the feature matures,
+this policy will change in favor of stronger security guarantees.
+
+### Getting started with TLS
+
+
+The conduit control plane must be installed with the `--tls=optional` flag:
+
+```
+conduit install --tls=optional |kubectl apply -f -
+```
+
+This causes a Certificate Authority (CA) container to be run in the control
+plane. The CA is responsible for watching the Kubernetes API for new
+Conduit-enabled pods and, as new pods are created, it ensures that the pod
```

Comment:
I suggest replacing "The CA is responsible..." with:

The CA watches for the creation and updates of Conduit-enabled pods. For each Conduit-enabled pod, it generates a private key, issues a certificate, and distributes the certificate and private key to each pod as a Kubernetes secret.